.github/workflows/format-check.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+    dprint-check:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v3
+        - uses: actions/setup-node@v3
+        - run: dirty=$(npx dprint check | grep "Found * not formatted file*.") >> "$GITHUB_STEP_SUMMARY"
```

Comment:
This needs an `npm install`, otherwise the version of dprint used will be unspecified which may cause problems if the plugins in the dprint config are too old to work with newer dprint versions.